unity-renderer/Assets/DCLServices/MapRendererV2/MapLayers/HomePoint/HomePointMarkerController.cs

```diff
@@ -67,6 +69,11 @@ public UniTask Disable(CancellationToken cancellationToken)
             return UniTask.CompletedTask;
         }
 
+        public void SetParameter(IMapLayerParameter layerParameter)
```

Comment:
Instead of having an empty method create a virtual method in `MapLayerControllerBase` and override it when necessary or you can do it directly in the interface

Reply:
presenting it on the base class would imply bigger refactoring (taking into account our down-cast from generic child of this interface). So I implemented default ctor in interface itself 

https://github.com/decentraland/unity-renderer/pull/5780/commits/d3cbefe70da9626a7919302e48dc4b84e323a327